### PR TITLE
[SPARK-33451][DOCS] Change to 'spark.sql.adaptive.skewJoin.skewedPartitionThresholdInBytes' in documentation

### PR DESCRIPTION
**Pull request by rivuletrain**
### What changes were proposed in this pull request?

In the 'Optimizing Skew Join' section of the following two pages:
1. [https://spark.apache.org/docs/3.0.0/sql-performance-tuning.html](https://spark.apache.org/docs/3.0.0/sql-performance-tuning.html)
2. [https://spark.apache.org/docs/3.0.1/sql-performance-tuning.html](https://spark.apache.org/docs/3.0.1/sql-performance-tuning.html)

The configuration 'spark.sql.adaptive.skewedPartitionThresholdInBytes' should be changed to 'spark.sql.adaptive.skewJoin.skewedPartitionThresholdInBytes', The former is missing the 'skewJoin'.

### Why are the changes needed?

To document the correct name of configuration

### Does this PR introduce _any_ user-facing change?

Yes, this is a user-facing doc change.

### How was this patch tested?

Jenkins / CI builds in this PR.

### CODE REVIEW

File: docs/sql-performance-tuning.md [hunk 1]
```diff
@@ -280,7 +280,7 @@ Data skew can severely downgrade the performance of join queries. This feature d
        <td><code>spark.sql.adaptive.skewJoin.skewedPartitionFactor</code></td>
        <td>10</td>
        <td>
-         A partition is considered as skewed if its size is larger than this factor multiplying the median partition size and also larger than <code>spark.sql.adaptive.skewedPartitionThresholdInBytes</code>.
+         A partition is considered as skewed if its size is larger than this factor multiplying the median partition size and also larger than <code>spark.sql.adaptive.skewJoin.skewedPartitionThresholdInBytes</code>.
```
Comment (janekdb) [inline docs/sql-performance-tuning.md:283]: LGTM

**Comment by HyukjinKwon** [comment]
@aof00, I fixed the PR title and description but please keep the GitHub PR template and fix the title according to http://spark.apache.org/contributing.html next time.

**Comment by HyukjinKwon** [comment]
Merged to master and branch-3.0.

**Review by HyukjinKwon — APPROVED** [review]

**Comment by HyukjinKwon** [comment]
Merged to master and branch-3.0.

**Comment by aof00** [comment]
> @aof00, I fixed the PR title and description but please keep the GitHub PR template and fix the title according to http://spark.apache.org/contributing.html next time.

OK, I'll remember that next time! Thanks!

**Pull request closed without merging**